cloudigrade/api/clouds/aws/tasks/cloudtrail.py

```diff
@@ -81,9 +81,10 @@ def analyze_log():
             aws.delete_messages_from_queue(queue_url, [message])
             successes.append(message)
         else:
-            log_message, log_args = _(
-                "Failed to process message id %(message_id)s; leaving on queue."
-            ), {"message_id": message.message_id}
+            log_message, log_args = (
+                _("Failed to process message id %(message_id)s; leaving on queue."),
+                {"message_id": message.message_id},
+            )
```

Comment:
What exactly prompted the this change? The old form was considered valid when flake8/black ran on it [just an hour ago](https://github.com/cloudigrade/cloudigrade/actions/runs/957636209).

Reply:
running poetry run tox -e flake8 locally wasn't happy somehow.

Reply:
To leave a breadcrumb since this was quietly resolved elsewhere, the cause was out-of-date local dependencies. Forcing a refresh with `tox -r` on Alberto's side made the previous version of the code valid again.

Reply:
summary: had imcompatible flake8 between the cached .tox one and the one from the poetry shell, tox -r -e flake8 did the trick.

Reply:
Must have been typing at the same time 😄